src/views/SystemProfile/components/SubSystems/index.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardBody,
+  CardGroup,
+  CardHeader,
+  Grid,
+  GridContainer,
+  Link
+} from '@trussworks/react-uswds';
+import classnames from 'classnames';
+
+import BookmarkCardIcon from 'components/BookmarkCard/BookmarkCardIcon';
+import UswdsReactLink from 'components/LinkWrapper';
+import {
+  DescriptionDefinition,
+  DescriptionTerm
+} from 'components/shared/DescriptionGroup';
+import Divider from 'components/shared/Divider';
+import SectionWrapper from 'components/shared/SectionWrapper';
+import useCheckResponsiveScreen from 'hooks/checkMobile';
+import { GetCedarSystems_cedarSystems as CedarSystemProps } from 'queries/types/GetCedarSystems';
+
+import './index.scss';
+
+type SubSystemsProps = {
+  system: CedarSystemProps;
+};
+
+type SubSystemDataTodo = {
+  id: string;
+  name: string;
+  acronym: string;
+  description: string;
+  retirementDate: string;
+};
+
+const subSystemsDataTodo: SubSystemDataTodo[] = [
+  {
+    id: '1',
+    name: 'Test Ocular Fiction Utensil',
+    acronym: 'TOFU',
+    description: 'Lorem ipsum description',
+    retirementDate: 'No planned retirement or replacement'
+  },
+  {
+    id: '2',
+    name: 'Bio-Energy Engagement File',
+    acronym: 'BEEF',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vitae, ut in pellentesque eget elementum malesuada velit magna. Bibendum in feugiat vitae egestas turpis augue. Vel quam donec magna eget aliquam vitae tempor, orci.',
+    retirementDate: 'Planned retirement: Q4 2022'
+  }
+];
+
+const pointOfContactDataTodo = {
+  name: 'Geraldine Hobbs',
+  role: 'Business Owner',
+  email: 'mailto:todo'
+};
+
+const SystemSubSystems = ({ system }: SubSystemsProps) => {
+  const { t } = useTranslation('systemProfile');
+  const isMobile = useCheckResponsiveScreen('tablet');
+  return (
+    <div id="system-sub-systems">
+      <GridContainer className="padding-left-0 padding-right-0">
+        <Grid row gap>
+          <Grid desktop={{ col: 8 }}>
+            <h2 className="margin-top-0 margin-bottom-4">
+              {t('singleSystem.subSystems.header')}
+            </h2>
+            <CardGroup className="margin-0">
+              {subSystemsDataTodo.map(sub => {
+                return (
+                  <Card key={sub.id} className="grid-col-12 margin-bottom-2">
+                    <CardHeader className="padding-2 padding-bottom-0">
+                      <div className="display-flex margin-bottom-105">
+                        <div className="flex-fill">
+                          <h3 className="margin-y-0 line-height-body-2">
+                            <UswdsReactLink to="todo">
+                              {sub.name}
+                            </UswdsReactLink>
+                          </h3>
+                        </div>
+                        <div className="flex-auto">
+                          <BookmarkCardIcon
```

Comment:
The bookmark icon still uses FA icon library, and does not play well USWDS utility classes.  To make the icon match the size in Figma, I found that creating a class and defining the font-size in scss will correctly adjust size.  We are moving away from FA shortly, so this isn't a huge deal.